crates/telio-nurse/src/heartbeat.rs

```diff
@@ -1199,4 +1208,19 @@ mod tests {
         assert_eq!(true, meshnet_enabled);
         assert_eq!("vpn:7600617f9f9db5691a8c2768bd9d8110:2", external_links);
     }
+
+    #[tokio::test]
+    async fn test_send_analytics_report_once_on_skipped_ticks() {
```

Comment:
Maybe use a paused time and advance the time [manually](https://docs.rs/tokio/latest/tokio/time/fn.advance.html)?

Reply:
Good point. After my latest changes does not seems necessary now.